gojira.sh

```diff
@@ -564,21 +565,27 @@ function image_name {
     OPENRESTY=${OPENRESTY:-$(yaml_find $yaml_file OPENRESTY)}
     LUAROCKS=${LUAROCKS:-$(yaml_find $yaml_file LUAROCKS)}
     OPENSSL=${OPENSSL:-$(yaml_find $yaml_file OPENSSL)}
+    BORINGSSL=${BORINGSSL:-$(yaml_find $yaml_file BORINGSSL)}
   fi
 
-  if [[ -z $LUAROCKS || -z $OPENSSL || -z $OPENRESTY ]]; then
+  if [[ -z $LUAROCKS || (-z $OPENSSL && -z $BORINGSSL) || -z $OPENRESTY ]]; then
```

Comment:
same as suggested previously

```suggestion
  if [[ -z $LUAROCKS || -z "${OPENSSL}${BORINGSSL}" || -z $OPENRESTY ]]; then
```